content/en/logs/workspace/_index.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Logs Workspace
+kind: documentation
+disable_toc: false
+---
+
+{{< callout url="#" header="false" >}}
+  Logs Workspace is in private beta.
+{{< /callout >}}
+
+## Overview
+During an incident investigation, you might need to run complex queries, such as combining attributes from multiple log sources or transforming log data, to analyze your logs. Use Logs Workspace to run queries to:
+
+- Correlate multiple data sources
+- Aggregate multiple levels of data
+- Join data using SQL
+- Extract data or add a calculated field at query time
+
+You can also add visualizations for your transformed datasets.
+
+## Create a workspace and add a data source
+
+You can create a workspace from the Workspaces page or from the Log Explorer.
+
+On the [Logs Workspaces][1] page:
+
+1. Click **New Workspace**.
+1. Click the **Data source** tile.
+1. Enter a query. The reserved attributes of the filtered logs are added as columns.
+
+In the [Log Explorer][2]:
+
+1. Enter a query.
+1. Click **More** and select **Open in Workspace**.
+1. The workspace adds the log query to a data source cell. By default, the columns in Log Explorer are added to the data source cell.
+
+## Analyze, transform, and visualize your logs
+You can add the following cells to include additional data sources such as reference tables, use SQL to join data, and also transform, correlate, and visualize the data. Cells that depend on other cells are automatically updated when one of the cells it depends on is changed.
+
+Click the dataset on the left side of the page to go directly to that cell.
+
+### Data source cell
+
+You can add a logs query or a reference table as a data source.
+
+1. Click on the **Data source** tile.
```

Comment:
@estherk15 we were looking at notebooks and it uses cells on this page: https://docs.datadoghq.com/notebooks/

I'll leave as cells for now because a "+ Add cell" button pops up when you hover between cells.